wandb/sdk/wandb_artifacts.py

```diff
@@ -1743,7 +1736,7 @@ class WBArtifactHandler(StorageHandler):
     _client: Optional[PublicApi]
 
     def __init__(self, scheme: Optional[str] = None) -> None:
-        self._scheme = scheme or "wandb-artifact"
```

Comment:
I know we don't use the `scheme` override, but what was the rationale for removing it?

Reply:
rational: we use `self._scheme` in the `store_path` method and i think ideally is just static - no need to override.